dotcom-rendering/src/server/lib/logging-store.ts

```diff
@@ -19,6 +19,7 @@ type DCRLoggingStore = {
 		type?: string;
 		platform?: string;
 	};
+	timing: { [key: string]: number };
```

Comment:
I think that having explicit start and stop times will make reasoning about this easier, rather than relying on the value being set or not.

```suggestion
	timing: { [key: string]: { start: number, stop?: number } };
```

Reply:
Possibly, although I think we'd still want some kind of "duration" property though since it'll be harder to do the maths in Elasticsearch / Grafana

Reply:
yeah a duration is what you want to to log, but whilst the data isn’t finalised, you probably want `startTime` and `duration`, much like a [`PerformanceMeasure`](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceMeasure)